resources/ci_cd/jenkins_output.py

```diff
@@ -46,22 +46,33 @@ def main():
 
     if platform.system() == "Linux":
         gcc_version = (
-            subprocess.run(["gcc", "-dumpfullversion"], capture_output=True)
+            subprocess.run(
+                ["gcc", "-dumpfullversion"],
+                capture_output=True,
+            )
             .stdout[:-1]
             .decode("utf-8")
+            or "-"
         )
-        extra_rows = f"| GCC Version | {gcc_version} |\n"
 
+        extra_rows = f"| GCC Version | {gcc_version} |\n"
         clang_version = (
-            subprocess.run(["clang", "-dumpversion"], capture_output=True)
+            subprocess.run(
+                ["clang", "-dumpversion"], capture_output=True, check=True
+            )
             .stdout[:-1]
             .decode("utf-8")
+            or "-"
         )
+
         extra_rows = f"| CLANG Version | {clang_version} |\n"
```

Comment:
Are you overwriting the variable extra_rows here? Maybe you meant:

```suggestion
        extra_rows = f"{extra_rows}| CLANG Version | {clang_version} |\n"
```

Reply:
Good catch!